CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 
+- `[jest-cli]` Only set error process error codes when they are non-zero [#7363]
```

Comment:
Should include full url as well, so the link works outside of GitHub

Reply:
Updated.